Frontend/ui-customer/src/Components/ContactList/View.purs

```diff
@@ -0,0 +1,285 @@
+        , stroke $ "1," <> Color.grey900
+        , alignParentBottom "true,-1"
+        , margin (Margin 0 0 0 0)
+        ]
+        [ linearLayout
+            [ width MATCH_PARENT
+            , height if os == "IOS" then (V 52) else WRAP_CONTENT
+            , gravity BOTTOM
+            , alignParentBottom "true,-1"
+            ]
+            [ PrimaryButton.view (push <<< PrimaryButtonActionController) (primaryButtonConfig config.count)
+            ]
+        ]
+    ]
+
+genericHeaderConfig :: ContactsState -> GenericHeaderConfig.Config
+genericHeaderConfig state =
+  let
+    config = GenericHeaderConfig.config
+
+    genericHeaderConfig' =
+      config
+        { height = WRAP_CONTENT
+        , prefixImageConfig
+          { height = V 25
+          , width = V 25
+          , imageUrl = "ny_ic_chevron_left,https://assets.juspay.in/nammayatri/images/common/ny_ic_chevron_left.png"
+          , margin = (Margin 12 12 12 12)
+          }
+        , padding = (Padding 0 5 0 5)
+        , textConfig
+          { text = ((show (state.count)) <> "/" <> show (3 - (length state.contactList)) <> " " <> (getString CONTACTS_SELECTED))
+          , textSize = FontSize.a_18
+          , color = Color.darkDescriptionText
+          , fontStyle = FontStyle.semiBold LanguageStyle
+          }
+        , suffixImageConfig
+          { visibility = GONE
+          }
+        }
+  in
+    genericHeaderConfig'
+
+horizontalLine :: Int -> Int -> forall w. PrestoDOM (Effect Unit) w
+horizontalLine marginLeft marginRight =
+  linearLayout
+    [ height $ V 1
+    , width MATCH_PARENT
+    , background Color.grey900
+    , margin (Margin marginLeft 0 marginRight 0)
+    ]
+    []
+
+showEmergencyContact :: forall w. (Action -> Effect Unit) -> ContactsState -> PrestoDOM (Effect Unit) w
+showEmergencyContact push config =
+  scrollView
+    [ width MATCH_PARENT
+    , height MATCH_PARENT
+    , background Color.blue600
+    , weight 1.0
+    ]
+    [ linearLayout
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        , background Color.blue600
+        , margin (Margin 0 0 0 0)
+        ]
+        (showEmergencyContactData push config)
+    ]
+
+showEmergencyContactData :: forall w. (Action -> Effect Unit) -> ContactsState -> Array (PrestoDOM (Effect Unit) w)
+showEmergencyContactData push config =
+  ( mapWithIndex
+      ( \ind1 letter ->
+          ( linearLayout
+              [ height MATCH_PARENT
+              , width MATCH_PARENT
+              , orientation VERTICAL
+              ]
+              [ linearLayout
+                  [ height WRAP_CONTENT
+                  , width MATCH_PARENT
+                  , orientation HORIZONTAL
+                  , padding (Padding 16 12 0 12)
+                  , background Color.blue600
+                  , visibility if (((filter (\contact -> DS.take 1 contact.name == letter) config.contactsData) /= []) && (config.editedText == "")) then VISIBLE else GONE
```

Comment:
if you will use `contains` then modify this as well

Reply:
Here we need only first letter to check so i am using filter.